### PR TITLE
Toast for each account creation steps

### LINKED ISSUE #199: While waiting for account to be ready, show alert messages at each bit of progress
(Account generation complete + after each tx.)

### LINKED ISSUE #200: Remove info and success prefixes from non-error/warn toasts

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#199 
#200 

### Auto-close the issue?

Closes #199 
Closes #200 

### Types of changes

New feature (non-breaking change that adds functionality)

### Notes

I've made a tiny performance improvement running `marketplace approval` and `transfer mana` transactions in parallel.

### CODE REVIEW

File: decentraland/screens/EthereumSignUpScreen.js [hunk 1]
```diff
@@ -21,12 +22,16 @@ export class EthereumSignUpScreen extends React.Component {
...
       setAccount(account);
+      showInfo(`Account generated`);
 
       const { address: accountAddress } = account;
 
-      await fundAccount(accountAddress);
-      await approveManaSpending(account);
-      showInfo(`Account created and funded!`);
+      await fundAccountWithEthers(accountAddress);
+      showInfo(`Account funded with ethers`);
```
Comment (pcowgill) [inline decentraland/screens/EthereumSignUpScreen.js]: I feel like `ETH` might be more familiar to users than `ethers`

Comment (pcowgill) [reply decentraland/screens/EthereumSignUpScreen.js]:
```suggestion
      showInfo(`Account funded with ETH`);
```

File: decentraland/screens/EthereumSignUpScreen.js [hunk 2]
```diff
@@ -21,12 +22,16 @@ export class EthereumSignUpScreen extends React.Component {
...
+      const fundWithMana = fundAccountWithMana(accountAddress);
+      const approveMarketplace = approveManaSpending(account);
+      await Promise.all([fundWithMana, approveMarketplace]);
```
Comment (pcowgill) [inline decentraland/screens/EthereumSignUpScreen.js]: Is it an issue if an approve transaction happens before the account has MANA?

File: decentraland/screens/EthereumSignUpScreen.js [hunk 3]
```diff
@@ -21,12 +22,16 @@ export class EthereumSignUpScreen extends React.Component {
...
+      showInfo(`The account is ready to buy!`);
```
Comment (pcowgill) [inline decentraland/screens/EthereumSignUpScreen.js]: "buy" is usually a transitive verb, so it's mostly used like this: "buy `[SOMETHING]`".

Comment (pcowgill) [reply decentraland/screens/EthereumSignUpScreen.js]:
```suggestion
      showInfo(`Now you can buy land!`);
```

File: decentraland/screens/EthereumSignUpScreen.js (hunk 2 above)
Comment (pcowgill) [reply decentraland/screens/EthereumSignUpScreen.js]: > I've made a tiny performance improvement running `marketplace approval` and `transfer mana` transactions in parallel.

This question is related to your note

Comment (marcelomorgado) [reply decentraland/screens/EthereumSignUpScreen.js]: No, not really. The `approve` function doesn't check the `msg.sender` balance.

Comment (pcowgill) [reply decentraland/screens/EthereumSignUpScreen.js]: Hmm, that's weird. Good to know. Okay, resolving this and we can keep that in mind.

Comment (marcelomorgado) [reply decentraland/screens/EthereumSignUpScreen.js]: I've looked `MANAToken`  and `OpenZeppelin` ERC20/ERC721 codes and none of them require balance.

Comment (pcowgill) [reply decentraland/screens/EthereumSignUpScreen.js]: Thanks. Yep, I checked OpenZeppelin too. K, good to know!

**Pull request merged**